src/index.ts

```diff
@@ -160,31 +145,31 @@ export default function (
             $('#pdfLinkSidebarMenu').append(printMenuContent);
           }
 
-          const checkAndInsertPdfButtons = function() {
+          const insertPdfButtons = function() {
+            var pdfDownloadButton = $('' +
+            '<div class="navbar__item dropdown dropdown--hoverable dropdown--right" id="pdfDownloadMenu">' +
+            '  <a class="navbar__item navbar__link pdfLink" id="pdfLink" href="#">${pluginOptions.downloadButtonText}</a>' +
+            '  <ul class="dropdown__menu" id="pdfDownloadMenuList"></ul>' +
+            '</div>');
+            $(".navbar__items--right").prepend(pdfDownloadButton);
 
-            if ( !$("#pdfLink").length ) {
-              var pdfDownloadButton = $('' +
-              '<div class="navbar__item dropdown dropdown--hoverable dropdown--right" id="pdfDownloadMenu">' +
-              '  <a class="navbar__item navbar__link pdfLink" id="pdfLink" href="#">${pluginOptions.downloadButtonText}</a>' +
-              '  <ul class="dropdown__menu" id="pdfDownloadMenuList"></ul>' +
-              '</div>');
-              $(".navbar__items--right").prepend(pdfDownloadButton);
+            $("#pdfDownloadMenu").mouseenter(fillDownloadDropdownMenu);
 
-              $("#pdfDownloadMenu").mouseenter(fillDownloadDropdownMenu);
-            }
-
-            if (!$("#pdfLinkSidebar").length) {
-              var pdfDownoadButtonSidebar = $('<li class="menu__list-item menu__list-item--collapsed" id="pdfLinkSidebar"><a role="button" class="menu__link menu__link--sublist">${pluginOptions.downloadButtonText}</a><ul class="menu__list" id="pdfLinkSidebarMenu" style=""></ul></li>');
-              $('.navbar-sidebar__items > .menu > .menu__list').append(pdfDownoadButtonSidebar);
-              $('#pdfLinkSidebar').click(function() {
-                $('#pdfLinkSidebar').toggleClass('menu__list-item--collapsed');
-              });
-              $('.navbar__toggle').click(fillDownloadSidebarMenu);
-            }
+            var pdfDownoadButtonSidebar = $('<li class="menu__list-item menu__list-item--collapsed" id="pdfLinkSidebar"><a role="button" class="menu__link menu__link--sublist">${pluginOptions.downloadButtonText}</a><ul class="menu__list" id="pdfLinkSidebarMenu" style=""></ul></li>');
+            $('.navbar-sidebar__items > .menu > .menu__list').append(pdfDownoadButtonSidebar);
+            $('#pdfLinkSidebar').click(function() {
+              $('#pdfLinkSidebar').toggleClass('menu__list-item--collapsed');
+            });
+            $('.navbar__toggle').click(fillDownloadSidebarMenu);
           }
   
           $(window).on('load', function () {
-            setInterval(checkAndInsertPdfButtons, 1000);
```

Comment:
Maybe I broke something by removing this interval, but I couldn't figure out why it was running all the time every second?